### PR TITLE
Gefs fix

### PR DESCRIPTION
**Pull request by notAmine**

### CODE REVIEW

**Comment by arita37** [comment]
AUC is still incorrct....

Check 
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html

sklearn.metrics.roc_auc_score(y_true, y_score, *, average='macro', sample_weight=None, max_fpr=None, multi_class='raise', labels=None)[source]
Compute Area Under the Receiver Operating Characteristic Curve (ROC AUC) from prediction scores.

y_scorearray-like of shape (n_samples,) or (n_samples, n_classes)
PROBABILITY



https://github.com/AlCorreia/GeFs/blob/master/experiments/Outlier-Detection-Experiments.ipynb
# Predict outliers using GeFs
gef_red_pred = gef_red.log_likelihood(standardize_data(red_white, mean_red, std_red))
gef_white_pred = gef_white.log_likelihood(standardize_data(red_white, mean_white, std_white))

**Comment by arita37** [comment]
Auc numbers look reasonable.
